www/docs/en/dev/guide/platforms/windows/index.md

```diff
@@ -90,17 +84,31 @@ The tools and SDKs for the target Windows platforms (UWP, 8.1, etc.) must also b
 
 After installation, you should be ready to develop apps targetting Windows platform. Refer to [Create your first app](../../cli/index.html) guide for details.
 
-By default the `cordova build` command produces two packages: Windows 8.1 and Windows Phone 8.1.
-To upgrade Windows package to version 10 the following configuration setting must be
-added to configuration file (`config.xml`).
+By default the `cordova build` command produces one package for Windows 10.
+
+#### Target version configuration
+
+If you want to build Windows 8.1 and Windows Phone 8.1 packages by default, you have to target `8.1` and the following configuration setting must be added to configuration file (`config.xml`).
 
 ```xml
-<preference name="windows-target-version" value="10.0" />
+<preference name="windows-target-version" value="8.1" />
+```
+
+Once you add this setting, the `build` command will start producing Windows 8.1 and Windows Phone 8.1 packages.
+
+#### Overriding configuration with the --appx parameter
+
+You may decide that you want to build a particular version of your application targeting a particular OS (for example, you might have set that you want to target Windows 10, but you want to build for Windows Phone 8.1).  To do this, you can use the `--appx` parameter:
+
 ```
+cordova build windows -- --appx=8.1-phone
+```
+
+The build system will ignore the preference set in config.xml for the target Windows version and strictly build a package for Windows Phone 8.1.
 
-Once you add this setting `build` command will start producing Windows 10 packages.
+Valid values for the `--appx` flag are `8.1-win`, `8.1-phone`, and `uwp` or `uap` (for Windows 10 Universal Apps / Universal Windows Apps).  These options also apply to the `cordova run` command.
```

Comment:
Maybe convert this to a table?? Other then that this is OK still.

Reply:
Not worth the effort as all the 8.1 stuff will be gone soon - and as such probably the complete `appx` param. Otherwise, yeah.